CHANGELOG.md

```diff
@@ -4,6 +4,30 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 7.16.0
+
+This release adds the `withSentryConfig` feature to the Svelte SDK. It replaces the now deprecated Svelte `componentTrackingPreprocessor` which will be removed in the next major release.
+
+- feat(node): Export Span type from `@sentry/types` (#5982)
+- feat(svelte): Add `withSentryConfig` function to wrap User Svelte Configuration (#5936)
+- fix(nextjs): Correctly apply auto-instrumentation to pages in `src` folder (#5984)
+- fix(node): Improve logging in express scope tag match test (#5977)
+- fix(remix): Add yargs dependency for uploading sourcemaps (#5926)
+- fix(svelte): Track components without script tags (#5957)
+- fix(utils): Rename `global.ts` -> `worldwide.ts` (#5969)
+- fix(vue): Start pageload transaction earlier to capture missing spans (#5983)
+- ref(browser): Remove `getGlobalObject()` usage from `@sentry/browser` (#5884)
+- ref(build): Remove `constToVarPlugin` (#5970)
+- ref(core): Remove `getGlobalObject()` usage from `@sentry/core` (#5914)
```

Comment:
IMO let's remove all of the `getGlobalObject` PRs, they are pretty low value in the changelog and shouldn't have any user changes - same with the `constToVarPlugin` change.

We can maybe keep `Deprecate `getGlobalObject` as it's no longer used` as it will be useful for react native to know.

Reply:
Sure can do that. Makes totally sense for the `getGlobalObject` onces. I just thought I'd leave the `constToVarPlugin` PR in because it fixed a few Vue bugs

Reply:
> I just thought I'd leave the `constToVarPlugin` PR in because it fixed a few Vue bugs. So IMO it might be interesting for users despite it being a build change on our end

Yeah that's fair - we can leave it in then.